chore: bump dependencies

vitorhugods opened the pull request:
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [X] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [X] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [X] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

With coroutines 1.7.1, it seems now our tests that include ViewModel are properly running the `launch` inside the `init` block of the ViewModels. This is great, but as many of our tests were completely ignoring this, we had missing stubs.

### Solutions

Add the missing stubs when defining Mockks, so ViewModels can initialize properly.

### Dependencies

- https://github.com/wireapp/kalium/pull/1825

----
#### PR Post Merge Checklist for internal contributors

- [X] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

red-avtovo commented:
build again

red-avtovo commented:
Bug 1: timestamp is not updated automatically. Needed to restart the app
![766cb76e-8753-49fe-b37f-4130f5d2a709](https://user-images.githubusercontent.com/10682826/49090554-47c8f500-f25e-11e8-8f51-4ddedac3cd5a.jpg)

makingthematrix commented:
The timestamp update happens only when there is some activity in the conversation and the message with the timestamp gets recycled and displayed again. But this is also what happens with the timestamps on separators between messages, so I think it's sufficient.

red-avtovo commented:
lgtm

vitorhugods committed:
chore: bump dependencies

With coroutines 1.7.1, it seems now our tests that include ViewModel are properly running the `init` block of the ViewModels. This is great, but it means that many of our tests were completely ignoring this, so I had to add the missing stubs when defining Mockks.

vitorhugods committed:
Merge branch 'develop' into chore/bump-dependencies

vitorhugods committed:
style: remove unused import

vitorhugods committed:
Merge remote-tracking branch 'origin/chore/bump-dependencies' into chore/bump-dependencies

vitorhugods committed:
style: reduce line lengths

vitorhugods committed:
chore: bump composeCompiler to 1.4.7

vitorhugods committed:
chore: update kalium reference

AndroidBob commented:
Build [94](https://10.10.124.134/job/AR-build-pipeline/94/) **failed**.

vitorhugods committed:
chore: bump kotlin to 1.8.21

AndroidBob commented:
Build [96](https://10.10.124.134/job/AR-build-pipeline/96/) **failed**.

vitorhugods committed:
fix: remove duplicated MANIFEST.MF file

vitorhugods committed:
chore: upgrade gradle to 7.6.1

vitorhugods committed:
Merge branch 'develop' into chore/bump-dependencies

# Conflicts:
#	buildSrc/src/main/kotlin/Dependencies.kt
#	kalium

alexandreferris commented on app/src/test/kotlin/com/wire/android/ui/authentication/login/sso/LoginSSOViewModelTest.kt:
```diff
@@ -78,10 +77,9 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.extension.ExtendWith
 
+// @ExtendWith(CoroutineTestExtension::class)
```
Should we remove it then?

vitorhugods replied:
Yeaaaaaaaaaaaah. But it's in the queue :(

I'll remove in the next PR that adds version catalog to the mix.

alexandreferris reviewed (approved):
pretty nice! just one tiny comment

AndroidBob commented:
Build [150](https://10.10.124.134/job/AR-build-pipeline/150/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.3.0(6916455)(6916455)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.3.0(6916455)(6916455)-staging-compat.apk)
